sonaranalyzer-dotnet/its/expected/akka.net/Akka.Cluster.Sharding-{A05C31E8-0246-46A1-B3BC-4D6FE7A9AA49}-S4487.json

```diff
@@ -0,0 +1,56 @@
+{
+"issues":  [
+{
+"id":  "S4487",
+"message":  "Remove this unread private field 'Instance' or refactor the code to use its value.",
+"location":  {
+"uri":  "sources\akka.net\src\contrib\cluster\Akka.Cluster.Sharding\ShardRegion.cs",
```

Comment:
@alban-auzeill - this is weird, as it is a public static field
```
            public static readonly GracefulShutdown Instance = new GracefulShutdown();
```
I tried to repro it (see [this test](https://github.com/SonarSource/sonar-dotnet/pull/2722/files#diff-c6763e46582b95a8f4c341a9d3517512R254)) but cannot..